cuda_next/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# 3.15 is the minimum for including the project with add_subdirectory.
+# 3.21 is the minimum for the developer build.
+cmake_minimum_required(VERSION 3.15)
+
+# FIXME Tests currently fail when this is defined to Release. Temporary WAR to validate CMake changes:
+set(CMAKE_BUILD_TYPE "")
```

Comment:
Before merging, we need to make sure that tests pass without this WAR.

Reply:
I will take a look, thank you for looking into this